### PR TITLE
create a pull_request_template.md for the project

### PR DESCRIPTION
**Pull request by mahadevan-sreenivasan**
When someone creates a pull request template to the modus-web-components repository, project contributors will automatically see the template's contents in the pull request body.

Please review and let me know if anything needs to be changed / added / removed.

References #378

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Pull Request Template
```
Comment (coliff) [inline pull_request_template.md:1]: minor thing really but I don't think this needs a 'Pull Request Template' header.

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```
Comment (coliff) [inline pull_request_template.md:16]: Item 4 looks out of place here, should this be 'Documentation update'?

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```
Comment (coliff) [inline pull_request_template.md:11]: I think this line should be removed. I think it was a placeholder from the template.

**Review by coliff — APPROVED** [review]

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+## Description
```
Comment (ryan-henness-trimble) [inline pull_request_template.md:1]: sorry it took me so long to get to this. I think we should make the .md files camelcase, like `Pull_Request_Template.md`. We also may want to add examples of different keywords the developer can use for referencing the issue number. `Fixes`, `References`, and `Closes` are all good to know about.

Comment (ryan-henness-trimble) [reply pull_request_template.md:1]: We may also want to note that the checklist doesn't necessarily need to be in the description of the PR, but is necessary for the developer to make sure they have completed the checklist themselves

Comment (ryan-henness-trimble) [reply pull_request_template.md:1]: What do you think about making this `PR_Description_Template.md`?

Comment (ryan-henness-trimble) [reply pull_request_template.md:1]: @coliff you have any thoughts on these as well?

**Comment by ryan-henness-trimble** [comment]
Can you also squash your commits down to one commit when the changes are all together?

File: pull_request_template.md (hunk 4 above)
Comment (coliff) [reply pull_request_template.md:1]: I think the filename should be all lower-case, I think that's considered a best practise (all other files in the repo are lowercase).

The template filename itself won't be displayed to most users (contributors) so we don't need to try and optimize the filename for reading the name of it. I think it should be in the `.github` directory like GitHub's docs have it.
pr-template-hidden-directory

Comment (mahadevan-sreenivasan) [reply pull_request_template.md:1]: @ryan-henness-trimble : Like @coliff  said, I followed the [documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) from github docs. 

Do you have a specific text or description in the PR for Fixes, References and Closes? 

@coliff  Sure I can move it to the .github folder.

Comment (ryan-henness-trimble) [reply pull_request_template.md:1]: actually this seems great! I think if we run into an issue where contributors are accidently closing issues when they should have just been 'referencing', then we can update this doc later

Comment (ryan-henness-trimble) [reply pull_request_template.md:1]: @mahadevan-sreenivasan hey! just wanted to ping to see if you still wanted to complete this

Comment (mahadevan-sreenivasan) [reply pull_request_template.md]: @ryan-henness-trimble : Apologies for the delay in getting this done. I have moved the pull_request_template.md file to the .github directory and have squashed all 3 commits into a single commit. Let me know if this is fine

Comment (ryan-henness-trimble) [reply pull_request_template.md]: @mahadevan-sreenivasan thanks so much for the contribution! 😄

**Review by ryan-henness-trimble — APPROVED** [review]

**Pull request merged**